File: Lista_Logica_Programacao/Java/Ex5.java
// Exercício 5 - Verificar se um número é par ou ímpar

import java.util.Scanner;

public class Ex5 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int numero = sc.nextInt();

        if (numero % 2 == 0) {
            System.out.println("É par");
        } else {
            System.out.println("É ímpar");
        }

        sc.close();
    }
}
/*
Anotações:

- % funciona igual no Java
- Precisa usar parênteses na condição do if
- Java exige chaves {} mesmo se for uma linha
- Estrutura é mais rígida que Python
*/

File: Lista_Logica_Programacao/Python/ex5.py
# Exercício 5 - Par ou Ímpar

numero = int(input("Digite um número: "))

# % retorna resto da divisão
if numero % 2 == 0:
    print("É par")
else:
    print("É ímpar")

# Anotações:
# - Usei % para pegar o resto da divisão
# - Se o resto for 0 significa que é divisível por 2
# - if em Python não usa parênteses obrigatórios
# - Precisa de dois pontos ":" depois do if